backend/shop/serializers.py

```diff
@@ -45,3 +45,9 @@ class GroupSerializer(ModelSerializer):
     class Meta:
         model = Group
         fields = ["id", "name"]
+
+
+class ProductSerializer(ModelSerializer):
+    class Meta:
+        model = Product
+        fields = "__all__"
```

Comment:
don't do this, you'll accidentally leak private data - `explicit is better than implicit` - https://stackoverflow.com/questions/64070128/zen-of-python-explicit-is-better-than-implicit